container/container.go

```diff
@@ -484,7 +484,9 @@ func copyEscapable(dst io.Writer, src io.ReadCloser, keys []byte) (written int64
 		nr, er := src.Read(buf)
 		if nr > 0 {
 			// ---- Docker addition
+			preservBuf := []byte{}
```

Comment:
yea, I just wanted to avoid all of the alloc's and appends as we work our way thru the input.